Caboodle/Sms/Sms.shared.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Sms
+    {
+        public static bool CanSendSms => GetCanSendSms();
```

Comment:
I think this API is simple enough just to use the public API in each platform - no need to have a property and the methods. BUT... that is preference. I personally like your way as it helps with maintenance and development, but this is not my repository. 😄 @jamesmontemagno @Redth , what do you think?

Reply:
Agreed, I would remove